Accolades now export their XP

TomatechGames opened the pull request:
Accolades now export their XP into the Tier property of NamedItemData

TomatechGames committed:
Accolades now export their XP

Accolades now export their XP into the Tier property of NamedItemData

taradinoc commented on BanjoBotAssets/Exporters/UObjects/AccoladeExporter.cs:
```diff
@@ -25,5 +25,15 @@ internal sealed class AccoladeExporter(IExporterContext services) : UObjectExpor
+        {
+            if(asset.GetOrDefault<FScalableFloat>("XpRewardAmount") is FScalableFloat xpAmount)
+            {
+                //this should probably be moved to its own property in an ccolade-specific version of NamedItemData
```
Indeed it should. :)

TomatechGames replied:
A new NamedItemData subclass has been created for Accolades, in the same format as other NamedItemData subclasses, and it includes an AccoladeXP property to hold XP Reward Amounts

TomatechGames committed:
Accolades have their own NamedItemData subclass

Accolades have their own NamedItemData subclass, with an AccoladeXP property used to hold XP amounts instead of Tier

TomatechGames committed:
Added copyright notice to AccoladeItemData
